src/EventStore/StreamFeed/Event.php

```diff
@@ -9,17 +8,53 @@
 class Event
 {
     /**
+     * @var integer
+     */
+    private $version;
+
+    /**
+     * @var string
+     */
+    private $type;
+
+    /**
      * @var array
      */
     private $data;
 
     /**
-     * @param array $data
-     * @param int   $version
+     * @var array
+     */
+    private $metadata;
+
+    /**
+     * @param array $content
+     */
+    public function __construct(array $content)
```

Comment:
what do you think about explicitly passing $version, $type and $data instead of an array?